CHANGELOG.md

```diff
@@ -6,7 +6,19 @@ The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
 
-## [0.1.0] — unreleased
+## [0.1.0] — 2020-05-12
+
+First release!
+
+### Added
+
+- Supports all instructions with the exception of floating points.
+- Passes majority of the official test suite. Does not implement all validation tests yet.
```

Comment:
That's a little stretch, we skip majority of the tests.

Reply:
The skipped ones are floating point and malformed source (not binary) tests.

Please suggest new wording, will apply. Not only for this line but the entire changelog.

Reply:
Updated.